src/api/protectedApiClient.ts

```diff
@@ -144,6 +153,44 @@ const getEventRegistrations: (eventId: number) => Promise<Registration[]> = (
   ).then((res) => res.data.registrations);
 };
 
+const eventCreate = (
+  request: NewEventInformation,
+): Promise<EventInformation> => {
+  return AppAxiosInstance.post(ProtectedApiClientRoutes.EVENTS, request)
+    .then((res) => res.data.id)
```

Comment:
Wait, so we only return the id (number) but this function signature says that we return an `EventInformation` object?

The TypeScript compiler doesn't detect a type mismatch here because of how it works: see https://stackoverflow.com/questions/40419804/typescript-not-throwing-error-on-wrong-promise-return-type for more info on why.

Regardless, either have it return a Promise<number> or fix line 160 to return `res.data`